### ISSUE TITLE
Marketing Goal: Become the default choice for YC teams

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#170]
### Context
This issue is mainly to help me plan and orchestrate work towards this goal, but feedback is still obviously welcome. 

One of my marketing goals for this quarter is to get 50% of the current W24 YC batch signed up to PostHog. I had this goal last quarter too, but it quickly became apparent that that goal was poorly timed and that initiatives took time to spin up.

The current batch, W24, starts in January 2024 and runs until March. There are **56** teams in the batch currently listed in the Startups Directory. 

As a baseline, [we claimed 20 teams in W23 from a batch of 273 teams](https://docs.google.com/spreadsheets/d/1v0lzSCNEc7StbsIbL1NxULxCFxFDUgFjrTFldJxp7NU/edit?usp=sharing), and we have 200 teams total from across all batches. 

The W24 seems to be much smaller (W23 batch was 273 teams) but applications for W24 may still be coming in even though the deadline is passed. Batches generally have been getting smaller, however. 

### What did we do in Q4?
Things we already did last quarter and are carrying forward, most of which are covered on [this issue](https://github.com/PostHog/company-internal/issues/1165). 

- Merged all YC batches into a central Slack channel, #posthog-founders-club
- Added a welcome bot to the channel to encourage interaction
- Started a cadence of weekly Slack posts from multiple authors to encourage engagement
- Weekly changelog updates into the channel to encourage adoption
- Adding YC teams to PostHog for Startups comms, and startup spotlights
- Adding some basic marketing polish to on-going activities, such as Kendall’s direct emails

We’ve trialled a few other tactics in the Slack channel, such as AMAs, but will discontinue these due to lack of engagement. Finally, we have jumped on ad-hoc opportunities, such as the Helicone x PostHog hackathon which was promoted to the YC community. 

### What’s new for Q1?
Here’s ideas for new stuff we could do this quarter. 

- [x] Startup spotlight focus on the most recent batch - https://github.com/PostHog/posthog.com/issues/7416
- [x] Give away new merch samples to YC & Startups orgs - https://github.com/PostHog/meta/issues/169
- [x] Trial a YC referral giveaway for selected founders - https://github.com/PostHog/company-internal/issues/1164
- [x] Follow up with Lindsay at YC about new video opportunities
- [x] Run a YC seminar with @jamesefhawkins and/or @timgl - https://github.com/PostHog/meta/issues/174
- [x] Make the YC merch giveaway a bit more generous (and include new merch)
- [x] ~Add a new merch item for everyday use~ - No longer planned due to merch issues 
- [x] Give away the PMF v2 books within YC campus 
- [ ] Give users an extra incentive if they add the PostHog badge to their site

An additional idea is a referral bonus run via Bookface, but I'm currently a little hazy on how we could set this up. 

### Feedback requested
More ideas are welcome. Improvements are welcome. Angry shouting is welcome. 

Would especially like feedback from @lottiecoxon about merch opps and what we can giveaway at a SF event (did the PMF guide work well, can we print more, how can we make it better?), and from @Lior539 @andyvan-ph @ivanagas and @charlescook-ph  on everything else.

**Comment by joethreepwood** [comment]
> Do you find that the Slack posts are worth continuing with? It seems like the channel is very support-y so you could drop that potentially and I don't think it would hurt things too much? (Especially if you're covering them via email updates anyway).

I'm not going to stop doing them, but I'm probably going to shove them down in the priority of things I do do. I have one ghost written for Lottie when the new merch goes live, for example, but I'll likely hold other things back and focus these around announcements.

**Comment by ivanagas** [comment]
1. Tell YC alumni/people we know in SF that we will sponsor their event targeting the new batch. Send merch, pizza, poster, etc.
2. Set up alerts for new companies announcing they are in the batch on Twitter/LinkedIn/YC site/HN/Product Hunt and like their posts, follow them, say congrats, etc. Maybe send them stuff?

**Comment by joethreepwood** [comment]
> 1. Tell YC alumni/people we know in SF that we will sponsor their event targeting the new batch. Send merch, pizza, poster, etc.

> 2. Set up alerts for new companies announcing they are in the batch on Twitter/LinkedIn/YC site/HN/Product Hunt and like their posts, follow them, say congrats, etc. Maybe send them stuff?

I love this idea. Challenge with sending them things may be that we don't have their addresses though. We could email them...but that's what @KendalHall already does with the invites.

@KendalHall maybe we could enhance those emails with an offer of some merch?

**Comment by lottiecoxon** [comment]
Concerning SF event merch:
- the PMF guide V2 is currently in draft, seeing as the remains of V1 was damaged in transit (I have [redesigned it to be more like a book instead of a thick leaflet](https://github.com/PostHog/company-internal/issues/1232)) but was waiting until the Hawk was back from paternity leave. I have reached out and will await his response. There was an idea thrown about to bulk it up with some key blog articles rewritten into some extra chapters/learning points, but also something to discuss. 
- Concerning other merch opportunities, I think learning from what has been successful in the past (stickers and hats) these would be a safe option to bring (also much less likely to get damaged in transit). I was inspired by our very own internal PostHog book club - we could include a custom book mark? Other ideas: glasses cloths, laptop sleeve, customised water bottles and super cool posters. Or we go way left field and make our own snakes and ladders mini set that goes with the book? Could be hedgehog themed or just have James and Tim falling down slides - unsure. 
- What is the set up at these events, I'm picturing a table where the merch will go? If this is the case how do we spruce it up. One thought would be set it up like a ironic book signing - but I'm worried people wouldn't see it as a joke, maybe not. Regardless we should get some kind of ikea mini shelves to display the items a bit more professionally - I could even make up some mini paper hogs to place around the table to add some fun. I think a giant banner would be predictable and a tad boring so would be keen to know what set up we would have so I can design something a bit more fun + left field.

**Comment by lottiecoxon** [comment]
Howdy, so printers for hardback books will take 3 weeks to produce, ship and arrive at destinations in SF. Prices range from $300-$700 depending if we want soft or hardback binding. If we can get content locked in asap then I can get drafts out to Joe, Cory, Andy etc (whoever else wants to look) for rigorous checks. 

TL;DR if we want these to arrive 4th of feb we should order it no later than the 12th Jan 2024.

**Comment by lottiecoxon** [comment]
Oh to add this would be for a run of 100 - would this be acceptable?

**Comment by joethreepwood** [comment]
Yep, 100 is perfect. We can reuse leftovers.

**Comment by joethreepwood** [comment]
We've ticked off all the items above now basically, but I want to keep testing ideas here regardless. Curious if anyone has feedback on these ideas - they feel a bit of a reach to me, but maybe better than doing nothing. 

- We target users in the _last_ batch. We send them a thank you gift and congrats for graduating the batch, see if it generates any WOM uplift. 
- We run a limited-time offer on our bookface deal. For the first two weeks of March, we offer users an additional incentive: extra $5,000 of credit, or an onboarding call with @jamesefhawkins to help them get set up. 
- We pursue a video interview with a YC partner we have access to, such as Dalton. We could record this from a live event and invite current YC users to dial in and pitch questions, then share the video on socials.

**Comment by joethreepwood** [comment]
I've taken silence as nobody feeling strongly about these ideas, which is fine. I've done [a bit of analysis](https://posthog.slack.com/archives/C0351B1DMUY/p1708102529314669) recently though and may be able to specifically target the users who have announced and who are not in our deal.

**Comment by charlescook-ph** [comment]
I think Dalton said previous to @jamesefhawkins he was up for doing a video?

**Comment by joethreepwood** [comment]
Closing this as it's essentially the end of quarter for me. 

We think this was a success! We captured 40% of the batch!

**Comment by charlescook-ph** [comment]
'Default' = over 50% right??
